src/components/Form.js

```diff
@@ -0,0 +1,52 @@
+import { useState, useRef } from 'react';
+import styled from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+
+export default function Form({ savedList, setSavedList }) {
+  const [input, setInput] = useState([]);
```

Comment:
초기값이 `[]` 빈 배열인 이유가 있나요? `input`에는 `String` 값만 들어오니 `''`로 설정하시는 게 맞을 것 같습니다.

```suggestion
  const [input, setInput] = useState('');
```